tests/virtualenv_tools_test.py

```diff
@@ -111,6 +111,49 @@ def test_move(venv, capsys):
     assert_virtualenv_state(venv.after)
 
 
+def test_move_installed_not_editable(tmpdir, capsys):
```

Comment:
this test name should probably indicate what it's regression testing: a non-utf8 script in bin

Reply:
The script itself is utf8, but py3 will try to read it as ascii if LANG isn't defined.

Reply:
err yes, I meant non-ascii, good catch :)